boot.sh

```diff
@@ -1,30 +1,37 @@
+#!/bin/bash
 # Boot up github shell access for Windows WSL/Ubuntu.
 
-#!/bin/bash
+if [ "$#" -lt 1 ]
+then
+echo  "Please insert at least one argument"
+exit
+fi
 
-private_key_location=~/.ssh/github_rsa # Change this location to where your file is located.
+myArray=( "$@" )
+private_key_location=${myArray[0]} 
+verbose=${myArray[1]}
 
-verbose=$1
+if [ ! -e $private_key_location ]; then
+  echo "key $private_key_location not found"
+  exit
+fi
 
 echo 'Starting...'
 
-if [ "$1" = "-v" ]; then
+if [ "$verbose" = "-v" ]; then
   echo 'Booting up ssh-agent...'
 fi
 
-eval `ssh-agent -s`
-# ssh_agent_running=$(ssh-agent -s)
+ssh_agent_running=$(ssh-agent -s)
```

Comment:
Can you test this once? I reckon that this might not work.

Reply:
yes the variable is storing the command output.

Reply:
Right, but the `ssh-agent -s` command actually outputs the ssh-agent's pid and the socket location and running the command inside `eval` stores them as local variables. If the `eval` is removed, `ssh-add` will no longer work, as shown below:

Reply:
![image](https://user-images.githubusercontent.com/13660453/103205563-d3bb1880-491f-11eb-9154-2a29c35204ab.png)